ceph_salt/validate/config.py

```diff
@@ -21,6 +21,12 @@ def validate_config(host_ls):
         dashboard_password = PillarManager.get('ceph-salt:dashboard:password')
         if not dashboard_password:
             return "No dashboard password specified in config"
+        dashboard_ssl_certificate = PillarManager.get('ceph-salt:dashboard:ssl_certificate')
+        dashboard_ssl_certificate_key = PillarManager.get('ceph-salt:dashboard:ssl_certificate_key')
+        if dashboard_ssl_certificate and not dashboard_ssl_certificate_key:
+            return "Dashboard SSH certificate provided, but no SSH certificate key specified"
+        if not dashboard_ssl_certificate and dashboard_ssl_certificate_key:
+            return "Dashboard SSH certificate key provided, but no SSH certificate specified"
```

Comment:
this code change mentions "SSH certificate" four times - should be "SSL certificate" in all cases I think

Reply:
Fixed